### ISSUE TITLE
Take advantage of local node disks for data that can be recreated

### ISSUE THREAD

**Issue by edevil** [colemickens/azure-kubernetes-status#16]
Since nodes have a fast local disk attached, wouldn't it be possible to store docker images and/or container filesystems in this disk? Upon a restart containers are recreated anyway and images can be fetched again.

**Comment by theobolo** [comment]
@edevil That was possible on Azkube, and tried to implement it on 'kubernetes-anywhere' (cf : https://github.com/kubernetes/kubernetes-anywhere/pull/227), it should definitely be available on ACS-Engine, by the way using a Premium Storage account on DS-Series gives really good perfs on the Docker layer. Maybe open a issue on ACS-Engine ?

**Comment by edevil** [comment]
Thanks! I'll take a look at the existing solutions to see if it is worth it.

Btw, @theobolo, do you usually run all your kubernetes nodes on the same storage account? Isn't there a best practice limit on the number of VMs?

**Comment by theobolo** [comment]
Yep it's something like 20 or 40 vhds/VMs per storage accounts if i remember well, but for the moment if i take the ACS Deployment example, it's deploying all the node Disks on the same Storage (and the Master OS disk on another).

I didn't try more than 10 nodes on my clusters, maybe that ACS is provisioning a second `Storage Account` when you deploy more than 20 nodes ? 

In that case perfs shouldn't be throttled by the Azure Storage Account limit, maybe that @colemickens knows. And shouldn't be a problem on cluster with less than 20 or 40 nodes :)

ps :  Founded that doc : https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits#virtual-machine-disk-limits
